packages/unplugin/src/index.ts

```diff
@@ -17,15 +19,12 @@ function generateGlobalInjectorCode({ release }: { release: string }) {
     _global.SENTRY_RELEASE={id:"${release}"};`;
 }
 
-export interface Options {
-  debugLogging?: boolean;
-}
-
 const unplugin = createUnplugin<Options>((options) => {
-  function debugLog(message: string) {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function debugLog(...args: any) {
     if (options?.debugLogging) {
       // eslint-disable-next-line no-console
-      console.log(`[Sentry-plugin] ${message}`);
+      console.log("[Sentry-plugin]}", args);
```

Comment:
nit: there is an additional `}` here

Reply:
Done [c5c5fec](https://github.com/getsentry/hackweek-sentry-unplugin/pull/23/commits/c5c5fec4619b1966f9c344392a0123b85da8b7ff)